<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginMenu</class>
 <widget class="QMainWindow" name="LoginMenu">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginMenu</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <property name="iconSize">
   <size>
    <width>10</width>
    <height>10</height>
   </size>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>700</width>
      <height>500</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="picture">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>350</width>
        <height>500</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/photos/nature2.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>0</y>
        <width>350</width>
        <height>500</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>0</y>
        <width>291</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;Ravie&quot;;</string>
      </property>
      <property name="text">
       <string>Create account</string>
      </property>
     </widget>
     <widget class="QLabel" name="account_picture">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>120</y>
        <width>40</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/photos/account.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="password_picture">
      <property name="geometry">
       <rect>
        <x>375</x>
        <y>190</y>
        <width>43</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/photos/password.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="account_signup">
      <property name="geometry">
       <rect>
        <x>432</x>
        <y>130</y>
        <width>201</width>
        <height>30</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>SizeVerCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 170, 255);
border:2px solid purple;
border-radius: 40px;



</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>32</number>
      </property>
      <property name="placeholderText">
       <string>Inser Username</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="password_signup">
      <property name="geometry">
       <rect>
        <x>432</x>
        <y>203</y>
        <width>203</width>
        <height>30</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>SizeVerCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 170, 255);
border:2px solid purple;
border-radius:40px;</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>32</number>
      </property>
      <property name="placeholderText">
       <string>Insert password</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="signup">
      <property name="geometry">
       <rect>
        <x>413</x>
        <y>280</y>
        <width>211</width>
        <height>61</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/photos/submit.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="toolTipDuration">
       <number>2</number>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/photos/submit.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 170, 255);
border:2px solid purple;
border-radius:20px;
font: 28pt &quot;Tempus Sans ITC&quot;;</string>
      </property>
      <property name="text">
       <string>Sign up</string>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="backtologin">
      <property name="geometry">
       <rect>
        <x>387</x>
        <y>430</y>
        <width>271</width>
        <height>21</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(128, 0, 128);
border:2px solid  rgb(255, 170, 255) ;
border-radius:50px;
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Back to Log-In</string>
      </property>
     </widget>
     <widget class="QLabel" name="signup_error">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>250</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;; color:red;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="design.qrc"/>
 </resources>
 <connections/>
</ui>
